tcms/core/views.py

```diff
@@ -65,6 +65,8 @@ def server_error(request):  # pylint: disable=missing-permission-required
     template = loader.get_template("500.html")
     return http.HttpResponseServerError(template.render({}, request))
 
+class SetupView(TemplateView):
+    pass
```

Comment:
You should try call_command() and perform the migrations here. See `migrations_order.py` and its parent classes for hints how to redirect output to the web.